### ISSUE TITLE
How to install?

### ISSUE THREAD

**Issue by gdebrion** [hollanbo/laravel-poeditor#1]
Hello,

I just install the package with `composer require hollanbo/laravel-poeditor` and it seems ok. But when I do `php artisan route:list` I don't see the ones of your module…

How to properly install your module?

Thanks :)

**Comment by hollanbo** [comment]
Hello,

I'm sorry for the late response, I've been very busy lately.
This package isn't really ready for use yet, that's why it has no installation instructions. I'll look into this problem over the weekend for you, though.

**Comment by gdebrion** [comment]
Ok, thanks :)

You are the only one project I found about editing po files in Laravel :)

**Comment by hollanbo** [comment]
ok, so I just went over this on a fresh install of laravel and as far as I can see, you just need to add the following line to your providers array in app.php:
`hollanbo\LaravelPoeditor\PoeditorProvider::class,`

Like I said, this package isn't even remotely ready for public use. I needed a poeditor for another project, so I wrote this. I hope it will work well for you and I'm sorry I don't have time to write proper documentation for it.

If you have any other problems or questions about this, feel free to ask and I'll try to answer ASAP.
